elements/pfelement/src/pfelement.js

```diff
@@ -207,18 +238,200 @@ class PFElement extends HTMLElement {
   }
 
   attributeChangedCallback(attr, oldVal, newVal) {
-    if (!this._pfeClass.cascadingAttributes) {
+    if (attr === "pfe-theme") {
```

Comment:
I think we'll need a more dynamic way to set this up because we'll have more than `theme` in the future for global attributes.

Reply:
Yeah, for sure.  I had the idea of giving the base class `static get globalProperties()` which would function exactly like `static get properties()`.  I didn't get around to implementing it yet though.